plugin/pick.vim

```diff
@@ -56,7 +60,7 @@ function! s:FileListCommand()
 endfunction
 
 function! s:GitListCommand(file)
-  return "git ls-files " . a:file . " --cached --exclude-standard --others"
```

Comment:
I prefer having untracked files showing up in pick.vim, and would very much like to keep it.

What's the performance loss in keeping it?